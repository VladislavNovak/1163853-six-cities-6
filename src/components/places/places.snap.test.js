import React from 'react';
import {render} from '@testing-library/react';

import {Places} from '..';

jest.mock(`../sorting-places/sorting-places`, () => `SortingPlaces`);
jest.mock(`../map/map`, () => `Map`);

test(`Should Places render correctly`, () => {
  const {container} = render(<Places />);
  expect(container).toMatchSnapshot();
});
