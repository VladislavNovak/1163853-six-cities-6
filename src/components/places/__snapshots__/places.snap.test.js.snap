// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Places render correctly 1`] = `
<div>
  <div
    class="cities"
  >
    <div
      class="cities__places-container container"
    >
      <sortingplaces />
      <map
        maptype="cities__map"
      />
    </div>
  </div>
</div>
`;
